src/Multi_shot/run.py

```diff
@@ -122,7 +121,8 @@ def attack_nlp_multishot(harmful_text: str, step: int, batch_prompts: List[str],
 
         writer.add_scalar("Evolution of score prompts Score", int(score), step)
         log_result(batch_number, harmful_text, victim_response, int(score))
-        return int(score)
+        scores.append(int(score))
+        return max(score)
```

Comment:
It's not iter, also you shouldn't have to return a max score cause in the case that there was a jailbreak before it will only output the maximum value for the rest of the queries.
Also it should be casted into an int due to the fact that the model outputs as a str